.github/workflows/trigger-dependent-repo-update.yml

```
@@ -20,10 +20,11 @@ jobs:
       - name: Trigger private repository workflow
         env:
           GH_TOKEN: "${{ steps.gen-token.outputs.token }}"
-          GH_REPO: cvat-ai/${{ secrets.CVAT_DEPENDENT_REPO }}
         run: |
-          gh workflow run \
-            update-dependency.yml \
-            -f triggeredBy=${{ github.repository }} \
-            -f commitSha=${{ github.sha }} \
-            -f branch=${{ github.ref }}
+          gh api \
+            --method POST \
+            /repos/cvat-ai/${{ secrets.CVAT_DEPENDENT_REPO }}/dispatches \
+            -f 'event_type=Update CVAT submodule' \
```

```suggestion
            -f 'event_type=upstream_updated' \
```

IMO, the event type should indicate only what happened to cause the event. It is up to the downstream repository to decide what to do (e.g. update the submodule). I'm also suggesting a more ID-like naming convention, like in the [example](https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#repository_dispatch).